bwapi/BWAPI/CMakeLists.txt

```diff
@@ -5,8 +5,11 @@ include_directories(
   ./Source
   ../BWAPICore
   ../Shared
+  ../OpenBWData
 )
 
+add_definitions(-DDEBUG_NEW)
```

Comment:
Can you remove this? It seems to be a msvc-specific thing and this doesn't seem to even be how you use it :P

Reply:
This is not even MS specific, but BWAPI specific define which intended to be used for `new` debugging (see https://github.com/OpenBW/bwapi/blob/develop-openbw/bwapi/Debug.h) if you would not mind, I better remove `#ifndef DEBUG_NEW` altogether

Reply:
Ah, I see. Is this defined (by default) in the vanilla BWAPI project? If so then I don't mind leaving it. As per Debug.h, it will only be used in a debug build anyways.

Reply:
So we are all set here ? or some additional comments?